### DISCUSSION TITLE
2.9.0 (2023-04-05)

### DISCUSSION THREAD

**Discussion by joshuajbouw** [aurora-is-near/aurora-engine#740] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/aurora-is-near/aurora-engine/releases/tag/2.9.0'>2.9.0 (2023-04-05)</a>.</em>

**Comment by birchmd** [comment]
Link to upgrade transaction on mainnet: https://explorer.mainnet.near.org/transactions/ChwMsrkCApna4w9pQe4tzfp2syJ5iUuvQbCXQCNZpDsP
